src/createCli.js

```diff
@@ -33,26 +34,41 @@ const createCli = (argv) => {
     .command({
       command: 'new',
       desc: 'Create new Graasp app.',
-      builder: (_) => _.option('s', {
-        alias: 'starter',
-        type: 'string',
-        default: DEFAULT_STARTER,
-        describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
-      }).option('f', {
-        alias: 'framework',
-        type: 'string',
-        describe: 'Set development framework (e.g. React, Angular)',
-      }).option('t', {
-        alias: 'type',
-        choices: ['app', 'lab'],
-        describe: 'Type of application (app or lab)',
-      }).option('p', {
-        alias: 'path',
-        type: 'string',
-        describe: 'Path where project directory will be set up.',
-      }),
+      builder: (_) =>
+        _.option('s', {
+          alias: 'starter',
+          type: 'string',
+          default: DEFAULT_STARTER,
+          describe: `Set starter. Defaults to ${DEFAULT_STARTER}`,
+        })
+          .option('f', {
```

Comment:
Tabs are a bit off.

Reply:
@juancarlosfarah This is actually the auto-format. I searched superficially for the option that is responsible for doing that but could not find it.

Reply:
If it's autoformat, then it's OK.